Realm/RLMSyncManager.h

```diff
@@ -79,14 +79,6 @@ typedef void(^RLMSyncErrorReportingBlock)(NSError *, RLMSyncSession * _Nullable)
 @property (nonatomic, copy) NSString *appID;
 
 /**
- Whether SSL certificate validation should be disabled. SSL certificate validation is ON by default. Setting this
```

Comment:
Deprecation notice is definitely a good idea, but the change I proposed doesn't seem to hard to do in the meantime to avoid entirely breaking existing uses of this in the wild?